tox.ini

```diff
@@ -57,24 +58,29 @@ skip_install = true
 deps =
   flake8
   flake8-2020
+  flake8-bandit
   flake8-blind-except
   flake8-bugbear
   flake8-builtins
+  flake8-coding
```

Comment:
If I understand correctly, the value to have this in addition to [a pre-commit hook](https://github.com/pre-commit/pre-commit-hooks#fix-encoding-pragma) is the check of the actual encoding value.
If my assumption is true, it would be nice to have this improvement in a separate PR with necessary changes to introduce the plugin for all repositories (`dependencies`, `stories`, `mappers`, `editors`). To be honest, I'm tired to keep repositories in sync manually. So please, do it in a way not to create additional load on me. I hope you understand.